ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -226,4 +235,12 @@ static boolean bug1162183Affected(Map<String, String> attributes) {
         && attributes.get(Attribute.APP_VERSION) != null
         && attributes.get(Attribute.APP_VERSION).matches("([0-3][0-9]|4[0-1])\\..*"); // 41 or older
   }
+
+  // See bug 1642386 for discussion of affected versions, etc.
+  static boolean bug1642386Affected(Map<String, String> attributes) {
+    return ParseUri.TELEMETRY.equals(attributes.get(Attribute.DOCUMENT_NAMESPACE))
+        && attributes.get(Attribute.DOCUMENT_TYPE).equals("sync")
+        && attributes.get(Attribute.APP_VERSION) != null
+        && attributes.get(Attribute.APP_VERSION).matches("(25|26)\\..*"); // 25 or 26
```

Comment:
```suggestion
        && attributes.get(Attribute.APP_VERSION).matches("^(25|26)\\..*"); // 25 or 26
```

I think `matches` will look for the pattern anywhere in the string, so we might need a `^` to explicitly match the beginning of the string.

Reply:
`matches` checks for the entire string to match. So the regex as it was wouldn't match something like 10.25.1. But it's probably best to be more explicit to avoid any confusion.